storm-core/src/native/worker-launcher/impl/worker-launcher.c

```diff
@@ -309,7 +309,7 @@ int set_user(const char *user) {
 }
 
 /**
- * Open a file as the node manager and return a file descriptor for it.
+ * Open a file as the worker-launcher user and return a file descriptor for it.
```

Comment:
Is this referring to "node manager" because it was copy pasted, or what is node manager referring to?

Maybe we should rename the functions that refer to "nm" as well?

Reply:
Possibly this code was copied from Hadoop.